From a6f6d11d42de34d0f6519c349f19217e269f6f56 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Fri, 26 Jan 2018 11:40:06 +0100
Subject: [PATCH] Fix invisible tray icon

It should call set_from_icon_name() instead of set_from_file(), otherwise it tries to set the icon from an invalid path.
---
 syncthing_gtk/statusicon.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/syncthing_gtk/statusicon.py b/syncthing_gtk/statusicon.py
index dc784ea..5f19ee9 100644
--- a/syncthing_gtk/statusicon.py
+++ b/syncthing_gtk/statusicon.py
@@ -240,7 +240,7 @@ def set(self, icon=None, text=None):
 	def _on_embedded_change(self, *args):
 		# Without an icon update at this point GTK might consider the icon embedded and visible even through
 		# it can't actually be seen...
-		self._tray.set_from_file(self._get_icon())
+		self._tray.set_from_icon_name(self._get_icon())
 		
 		# An invisible tray icon will never be embedded but it also should not be replaced
 		# by a fallback icon
